lib/utils/requireModule.js

```diff
@@ -1,12 +1,15 @@
 const path = require('path');
 
-const requireModule = module.exports = function(fullpath) {
+const requireModule = module.exports = function(fullpath, error={}) {
   let exported;
   try {
+    if (Object.keys(error).length > 0) {
```

Comment:
error should be a regular `Error` object and then you can do an `instanceof` check.

Reply:
fixed in [`4d6b137`](https://github.com/nightwatchjs/nightwatch/pull/3123/commits/4d6b1370eb894654388a2218c34a91b2c5d7b526)